.env.sample

```diff
@@ -1,4 +1,5 @@
 SKIP_PREFLIGHT_CHECK=true
+REACT_APP_LANGUAGE=en
```

Comment:
How the localization config is set, this must have an i18n JSON file in `src/configs/strings` or translations will fall back to English.